apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -469,23 +508,44 @@ function dragAndDrop(channel: 'inApp' | 'email' | 'sms', parent?: 'inApp' | 'ema
   const dataTransfer = new DataTransfer();
 
   cy.wait(1000);
-  cy.getByTestId(`dnd-${channel}Selector`).trigger('dragstart', { dataTransfer, force: true });
+  cy.getByTestId(`dnd-${channel}Selector`).trigger('dragstart', { dataTransfer });
   if (parent) {
-    cy.getByTestId(`node-${parent}Selector`).parent().trigger('drop', { dataTransfer, force: true });
+    cy.getByTestId(`node-${parent}Selector`).parent().trigger('drop', { dataTransfer });
   } else {
-    cy.getByTestId('node-triggerSelector').parent().trigger('drop', { dataTransfer, force: true });
+    cy.getByTestId('node-triggerSelector').parent().trigger('drop', { dataTransfer });
   }
 }
 function editChannel(channel: 'inApp' | 'email' | 'sms') {
-  cy.getByTestId(`node-${channel}Selector`).click({ force: true });
-  cy.getByTestId('edit-template-channel').click({ force: true });
+  cy.clickWorkflowNode(`node-${channel}Selector`);
+  cy.getByTestId('edit-template-channel').click();
 }
 
 function goBack() {
-  cy.getByTestId('go-back-button').click({ force: true });
+  cy.getByTestId('go-back-button').click();
 }
 
 function fillBasicNotificationDetails(title?: string) {
   cy.getByTestId('title').type(title || 'Test Notification Title');
   cy.getByTestId('description').type('This is a test description for a test title');
 }
+
+function waitLoadTemplate(beforeWait = (): string[] | void => []) {
```

Comment:
What do you think about some more generic names for this function? because I see that it waits for a bit more the just 'template'.

Reply:
Yes sounds more informative and understandable to me :)